### PR TITLE
Chsp latlon post

### PR DESCRIPTION
**Pull request by junwang-noaa**
In the commit:

1. add CHSP from GFSv16 package to gfsphysics and ccpp/physics
2. add capability to output global_latlon to netcdf file, new RT tests are added
3. add land surface model information to the history file, post lib is updated to 8.0.6
4. Add CCN-IN followup bugfix in GFS_typedefs.F90
5. Bugfix for Intel 18 on wcoss_cray, remove Intel 16 options for gaea
6. CMEPS integration code changes

Related PRs:
[ccpp/physics #440](https://github.com/NCAR/ccpp-physics/pull/440)
[fv3atm #104](https://github.com/NOAA-EMC/fv3atm/pull/104)
[NEMS #55](https://github.com/NOAA-EMC/NEMS/pull/55)
[#108](https://github.com/ufs-community/ufs-weather-model/pull/108)

### CODE REVIEW

**Comment by climbfuji** [comment]
I didn't see the newly added test `fv3_ccpp_wrtGlatlon_netcdf` in rt.conf. Is this intentional?

**Comment by junwang-noaa** [comment]
Those tests were committed to my local branch on dell, but I can not push
on dell and forgot about it, just push them on hera.

File: .gitmodules [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 	path = FV3
-	url = https://github.com/NOAA-EMC/fv3atm
-	branch = develop
+	url = https://github.com/junwang-noaa/fv3atm
+	branch = chsp_latlon_post
 [submodule "NEMS"]
 	path = NEMS
-	url = https://github.com/NOAA-EMC/NEMS
-	branch = develop
+	url = https://github.com/DeniseWorthen/NEMS
```
Comment (climbfuji) [inline .gitmodules]: Hasn't this been merged already?

Comment (climbfuji) [reply .gitmodules]: I see - just now. All good.

**Review by climbfuji — APPROVED** [review]

**Review by DusanJovic-NOAA — APPROVED** [review]

**Pull request merged**